src/main/scala/BintrayPlugin.scala

```diff
@@ -0,0 +1,194 @@
+package bintray
+
+import bintry.Attr
+import sbt.{ AutoPlugin, Credentials, Global, Path, Resolver, Setting, Task, Tags, ThisBuild }
+import sbt.Def.{ Initialize, setting, task, taskDyn }
+import sbt.Keys._
+import sbt.Path.richFile
+
+object BintrayPlugin extends AutoPlugin {
+  import BintrayKeys._
+  import InternalBintrayKeys._
+
+  override def requires = sbt.plugins.JvmPlugin
+  override def trigger = allRequirements
```

Comment:
If it's any consolation the reverse is also true: an auto-plugin doesn't auto-trigger you need to do something manually to each (or re-wrap in your own auto-plugin).

To _that_ problem we're considering an "always" trigger: https://github.com/sbt/sbt/issues/2093